executable/Env.re

```diff
@@ -59,7 +59,6 @@ let rec printUseOnCd = (~shell) =>
         if [[ -f .node-version && -r .node-version ]]; then
           echo "fnm: Found .node-version"
           fnm use
-          elsif
```

Comment:
I remember writing this, and I googled to see if this is how you do an `else if` on Bash, and realized that it is the Ruby syntax. 😳 
Great catch!

Reply:
@Schniz Thank you